yogstation/code/game/objects/items/premadepapers.dm

```diff
@@ -0,0 +1,53 @@
+//yogs code begins. Paperwork pre-made edition. What a pain this was. Most of these were taken from the wiki.
+/obj/item/paper/redd
```

Comment:
yeah, name them like `/obj/item/paper/paperwork` or something, this is just gonna be incredibly confusing for anyone looking through the object tree